packages/grid-html-export/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
You can just move this to `grid-export` package.

`grid-export` has a general name just for the purpose of having different types of export.

One argument against that is to minimize amount of dependencies (if you only use Excel, you don't want clipboard-polyfill), but the dependency is only 2.8 kB zipped and has no dependencies on its own.

Reply:
Maybe the opposite argument is more valid though, if you only need the copy-to-clipboard functionality and not export to excel; zipcelx is [96K](https://bundlephobia.com/result?p=zipcelx@1.6.2).

Reply:
Yeah maybe. We can see if there is a project with this requirement and extract it then.

Reply:
I have combined them now. What to do?
![image](https://user-images.githubusercontent.com/2663460/108382051-b6604780-7208-11eb-9970-96ee69dac954.png)